py3status/modules/volume_status.py

```diff
@@ -149,6 +143,97 @@ def current_volume(self):
         # get info about channel mute status
         muted = self._get_muted(output)
 
+        return perc, muted
+
+    def volume_up(self, delta):
+        # volume up
+        cmd = 'amixer -q -D {} sset {} '.format(self.device, self.channel)
```

Comment:
> Although I agree this is a valid concern I'm not sure if this is likely. I'm in favour of keeping things as simple as possible if we needed to refactor for such an event then fine but we could do it then, rather than having the complications and never using it.

I don't see how it's a complication. I understand that this means there are two `call` instead of one, but it makes sense to keep it in the backend code and I don't think it makes the code less readable.

> I just want nice easy readable code that is easy to maintain. where the call is made doesn't matter too much

In that case, I'd rather have it in the backend to avoid possible refactoring at a later date.